### DISCUSSION TITLE
Is there any way to move the window after extending the client area to title bar?

### DISCUSSION THREAD

**Discussion by hez2010** [AvaloniaUI/Avalonia#6013] [Q&A] [answered]
I set `ExtendClientAreaToDecorationsHint="True"` to extend the client area to title bar, but then I found I cannot move the window by dragging the window. Is there any way to achieve it?

Thanks.

**Comment by hez2010** [comment, marked as answer]
Solved with
```csharp
private void Window_PointerPressed(object sender, PointerPressedEventArgs args)
{
    var pointer = args.GetCurrentPoint(this);
    if (pointer.Position.Y < 20 && pointer.Pointer.Captured is Border)
    {
        BeginMoveDrag(args);
    }
}
```

**Reply by MarchingCube** [reply]
You can also take a look at the example in ControlCatalog. You need to set extended height hint and ensure that you don't have hit testable content in the titlebar. Then it should just work.
